bin/compile

```diff
@@ -39,7 +39,7 @@ $builder->buildFromIterator(
         ->files()
         ->name('*.php')
         ->name('*.pem*')
-        ->exclude(['Tests', 'tests', 'phpunit'])
+        ->exclude(['Tests', 'tests'])
```

Comment:
Because the PHAR needs `phpunit/phpunit/src/Framework/Assert/Functions.php`, autoloaded by Composer in the latest PHPUnit.